airflow/contrib/operators/mysql_to_gcs.py

```diff
@@ -135,24 +137,20 @@ def _write_local_data_files(self, cursor):
             names in GCS, and values are file handles to local files that
             contain the data for the GCS objects.
         """
-        class BinaryTypeEncoder(json.JSONEncoder):
-            def default(self, obj):
-                if PY3 and isinstance(obj, binary_type):
-                    return str(obj, 'utf-8')
-                return json.JSONEncoder.default(self, obj)
-
         schema = list(map(lambda schema_tuple: schema_tuple[0], cursor.description))
+        col_type_dict = self._get_col_type_dict()
```

Comment:
`col_type_dict` is only used in `self._convert_types(schema, col_type_dict, row)`, so I would rather pass it as a local variable